frontend/src/libs/components/profile-settings-sidebar/profile-settings-sidebar.tsx

```diff
@@ -0,0 +1,79 @@
+import { Card, Icon } from '#libs/components/components.js';
+import { IconColor } from '#libs/enums/enums.js';
+import { useAppSelector, useCallback, useState } from '#libs/hooks/hooks.js';
+import { type IconName, type SettingsOption } from '#libs/types/types.js';
+import { type UserAuthResponseDto } from '#packages/users/users.js';
+
+import styles from './styles.module.scss';
+
+const settingsOptions: SettingsOption[] = [
+  {
+    name: 'notification',
+    title: 'Very long text to see if it will break',
```

Comment:
rename name to key, use actual value for title for notification item and remove subscription option